### ISSUE TITLE
parameter documentation and scalar vs. per-species 

### ISSUE THREAD

**Issue by bernstei** [libAtoms/soap_turbo#1]
It would be nice if the regular SOAP / turbo-SOAP conversion table in the README was more explicit about what was optional (with default values) and what was required.

It would also be nice if the per-species parameters were assumed to apply to all species if only 1 value is specified.

**Comment by mcaroba** [comment]
I agree with you on both. Your second suggestion has been on my to-do list for a long time. I was working on improving the transferability between soap and soap_turbo strings a few months ago and it turned out it wasn't trivial, because of how the arrays are handled. That's not to say it can't be done, just that I need to budget some of my time to get it sorted. I should be able to fix that in May. I will sort out the README file sooner, since that's much more straightforward.
